packages/miniflare/src/workers/assets/rpc-proxy.worker.ts

```diff
@@ -0,0 +1,33 @@
+import { WorkerEntrypoint } from "cloudflare:workers";
+
+export default class SpecialDefault extends WorkerEntrypoint {
+	fetch(request) {
+		return this.env.ROUTER_WORKER.fetch(request);
+	}
+
+	constructor(...args) {
+		return new Proxy(super(...args), {
+			get(target, prop, receiver) {
+				// totally shit
+				if (prop === "fetch") {
+					const originalFetch = target[prop];
+					return function (...args) {
+						return originalFetch.apply(target, args);
+					};
+				}
+
+				if (prop === "env") {
```

Comment:
This seems like it exposes `env` over RPC, which doesn't seem like what you want?